src/software/ai/navigator/path_planner/path_planner_test.cpp

```diff
@@ -36,81 +36,71 @@ RobotNavigationObstacleFactory robot_navigation_obstacle_factory(
     DynamicParameters->getAIConfig()->getRobotNavigationObstacleFactoryConfig());
 
 std::vector<PlannerTestCase>
-    test_cases = {{.name               = "Empty field straight line",
-                   .start              = Point(0, 0),
-                   .end                = Point(1, 0),
-                   .navigable_area     = Rectangle({-1, -1}, {2, 2}),
-                   .obstacles          = {},
-                   .should_return_path = true},
+    test_cases =
```

Comment:
Can you explain the changes to these tests? They are failing in CI

Reply:
they might have been caused by #1661

Reply:
Yep sorry. These break on gcc 9.3 (the default ubuntu 20 compiler) because the call to brace enclosed initializer lists is ambiguous. 

I've reverted the changes since they should pass CI, even if they don't pass on my machine.